db/structure.sql

```diff
@@ -128,33 +114,6 @@ CREATE TYPE public.user_status_enum AS ENUM (
 );
 
 
---
--- Name: maptile_for_point(bigint, bigint, integer); Type: FUNCTION; Schema: public; Owner: -
---
-
-CREATE FUNCTION public.maptile_for_point(bigint, bigint, integer) RETURNS integer
-    LANGUAGE c STRICT
```

Comment:
why did you remove those functions?

Reply:
That's just a result of not having them in his database when the structure was dumped - when running migrations you need to carefully review changes to the structure and remove any that aren't appropriate before committing.